wuvt/defaults.py

```diff
@@ -30,7 +30,7 @@
 
 # DJ activity timer for automation/DJ logout in seconds
 DJ_TIMEOUT = 30 * 60
-EXTENDED_DJ_TIMEOUT = 120 * 60
+EXTENDED_DJ_TIMEOUT = 240 * 60
```

Comment:
Most DJs only have a live performance going on for one hour, two hours max. I've never seen one take an entire 2 hour show.